src/main/kotlin/com/terraformation/backend/search/field/AliasField.kt

```diff
@@ -44,6 +44,12 @@ private constructor(
    */
   override val nullable: Boolean = original.nullable || targetPath.sublists.any { !it.isRequired }
 
+  /**
+   * True if the target field can be exported to a CSV file. Delegate to original field for
+   * exportable check.
+   */
+  override val exportable: Boolean = original.exportable
```

Comment:
The class already delegates to the original (the `by` on line 27) for all properties/functions that aren't explicitly overridden. So you could remove this entirely and it'd do the right thing.